BAZProjectC1/View/Cells/MovieBasicInfoCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MovieBasicInfoCell.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 17/09/22.
+//
+
+import UIKit
+
+class MovieBasicInfoCell: UITableViewCell {
+
+    
+    @IBOutlet weak var movieTitle: UILabel!
+    @IBOutlet weak var moviePoster: UIImageView!
+    
+    static let  identificador = String(describing: MovieBasicInfoCell.self)
+    static func nib() -> UINib
+        {
+            return UINib(nibName: String(describing: MovieBasicInfoCell.self),bundle: nil)
+        }
```

Comment:
Esta función prodría ser privada?

Te hice algunas sugerencias de estilos:

```suggestion
    private static func nib() -> UINib {
            UINib(nibName: String(describing: MovieBasicInfoCell.self),bundle: nil)
    }
```